src/constants/lists.ts

```diff
@@ -5,14 +5,16 @@ export const OPTIMISM_LIST = 'https://static.optimism.io/optimism.tokenlist.json
 export const ARBITRUM_LIST = 'https://bridge.arbitrum.io/token-list-42161.json'
 export const POLYGON_LIST =
   'https://unpkg.com/quickswap-default-token-list@1.2.2/build/quickswap-default.tokenlist.json'
+export const CELO_LIST = 'https://raw.githubusercontent.com/celo-org/celo-token-list/main/celo.tokenlist.json'
```

Comment:
Is this available from any other endpoint? I don't think this is a CDN, so deploying it largely may cause user experience to suffer.

Reply:
nice catch, I recently deployed a statically hosted version

https://celo-org.github.io/celo-token-list/celo.tokenlist.json

replacing ...